Sources/Generics/GenericTableViewController.swift

```diff
@@ -223,6 +223,8 @@ private extension GenericTableViewController {
             return
         }
 
-        tableView.scrollToRow(at: IndexPath(row: indexToScrollTo, section: 0), at: .middle, animated: true)
+        DispatchQueue.main.async {
+            self.tableView.scrollToRow(at: IndexPath(row: indexToScrollTo, section: 0), at: .middle, animated: true)
```

Comment:
Dont' think this is a good idea because the scroll happens after the list page shows up, try to find out why 
```        self.tableView.scrollToRow(at: IndexPath(row: indexToScrollTo + 5, section: 0), at: .middle, animated: true)
```does not work for iOS 10, and come up with an alternative solution.

Reply:
Yeah, I had `animated: false` in wrong branch. Forgot to set it to `false` here